project-dashboard/scripts.js

```diff
@@ -0,0 +1,102 @@
+(async function () {
+  const graph = graphql('https://api.github.com/graphql', {
+    method: 'POST',
+    asJSON: true,
+    headers: {
+      'Authorization': 'bearer 7329b9bb56f49b69df5e3e1584be9d8bae847fa7'
```

Comment:
Let's put the PAT into a variable and keep it empty by default when it's checked in--we can fill it in locally when developing.